Makefile

```diff
@@ -22,16 +22,12 @@ testname ?= ./...
 # Remove files
 .PHONY: clean
 clean:
-	rm -f ./slack
-	rm -f ./slack-cli
-	rm -f ./hermes
 	rm -rf ./bin/
 	rm -rf ./dist/
 
 # Initialize project
 .PHONY: init
 init:
-	# Get the latest tags to build correctly
```

Comment:
🤔 I understand how this is useful, but in outputs it might be confused as a `TODO`

Some notation exists to "[stub](https://stackoverflow.com/a/71337660/7147271)" commands in a `Makefile` but I opted to delete this one instead for now 🪓